portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,28 @@
 #greeting-container {
   margin-top: 20px;
 }
+body{
+    padding: 0;
+    margin: 0;
+    position: relative;
+}
+span{
+    position: absolute;
+    width: 100vw;
+    background: rgb(245, 102, 102);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem;
+}
+span > *{
```

Comment:
I couldn't find this in the style guide but this selector says "for all elements directly within a `<span>` apply these rules". It's better to instead apply this to a specific element (that have an `id` attribute) or a class of elements (that have a `class` attribute)